Reactivate Portland 2020 tickets and announce dates (#1238)
writethedocs/www: docs/conf/portland/2020/news/announcing-dates-tickets.rst

> :template: {{year}}/generic.html

Reviewer: I'd feel a bit better if we renamed the file to "announcing-virtual-conf-dates-tickets" or something to be more specific..

Author: Done

.. post:: May 14, 2020
   :tags: portland-2020, speakers, tickets, visiting, sponsors

> Announcing ticket sales and dates

Reviewer: ```suggestion
Announcing virtual conference dates and ticket sales
```

=================================

> As we've :doc:`announced </blog/moving-portland-prague-2020-online>`, Write the Docs Portland 2020 will be a virtual conference.

Reviewer: ```suggestion
As we :doc:`announced </blog/moving-portland-prague-2020-online>`, Write the Docs Portland 2020 will be a virtual conference.
```

> We've decided to keep the dates as previously announced: the writing day is on

Reviewer: ```suggestion
We decided to keep the dates as previously announced: the writing day is on
```

**{{date.day_two.dotw}}, {{date.month}} {{date.day_two.date}}, 9am-6pm {{tz}}**,

> and the main conference days on **{{date.day_three.dotw}}, {{date.month}} {{date.day_three.date}}, 9am-6pm {{tz}}**

Reviewer: ```suggestion
and the main conference days are on **{{date.day_three.dotw}}, {{date.month}} {{date.day_three.date}}, 9am-6pm {{tz}}**
```

> We have reopened :doc:`ticket sales </{{shortcode}}/{{year}}/tickets>`. Your virtual ticket includes:

Reviewer: ```suggestion
**Ticket sales will re-open on Monday, May 25th**. Check out the :doc:`Tickets page </{{shortcode}}/{{year}}/tickets>` for prices. Your virtual ticket includes:
```

Reviewer: I feel like opening tickets on a Monday sounds better, and we also need to prepare more things around grants and whatever, so I figured it's better to have a few more days rather than too few ;)

> * Live streaming of all talks

Reviewer: ```suggestion
* Live streaming of all talks (available for ticket-holders only)
```

* Q&A with speakers after each talk (may not be available for all speakers)
* Access to the conference chat with all other attendees, speakers and sponsors
* Access to the writing day
* The virtual job fair

> If you have chosen to :doc:`previously bought ticket </conf/portland/2020/ticket-choices>` into a

Reviewer: ```suggestion
If you have chosen to convert your :doc:`previously bought ticket </conf/portland/2020/ticket-choices>` into a
```

virtual pass, or a pre-order for the virtual Portland 2020 conference, you will receive further
details by e-mail within the next week.

We are aiming to include all our speakers in the virtual conference, but still confirming
their availability.

> 

Reviewer: ```suggestion
Our opportunity grant will also re-open together with ticket sales in a modified format. Details will be published next week.
```
